docs/Guides/Scim/Guide-to-sync-users-in-vendasta.md

```diff
@@ -0,0 +1,52 @@
+# Guide to use Vendasta SCIM APIs to sync Users
+
+
+## Overview
+System for Cross-domain Identity Management ([SCIM](https://en.wikipedia.org/wiki/System_for_Cross-domain_Identity_Management)) focuses on syncing user accounts and permissions between systems but has an extension system that allows syncing any type of record. There are several well known endpoint paths for discovering what the server supports and resource schema definitions.ff
+
+This guide provides the information of
+
+- To create a service account in vendasta and authorization token to access vendasta APIs
```

Comment:
Provide link for already documented [Creating a service account](https://developers.vendasta.com/platform/fbf315c867e10-creating-a-service-account) and [Obtaining an Access Token for a service account](https://developers.vendasta.com/platform/ZG9jOjEwMTkzMDg5-obtaining-an-access-token-for-a-service-account) in this section

Reply:
that is given in authorization part , this is like a table of content what is covered in in this guide